README.md

```diff
@@ -132,9 +132,11 @@ of StorageOS.
 An example of of `StorageOSUpgrade` resource is [storageos_v1alpha1_storageosupgrade_cr.yaml](/deploy/crds/storageos_v1alpha1_storageosupgrade_cr.yaml).
 
 Only offline upgrade is supported for now by cluster-operator. During the
-upgrade, the applications that use StorageOS volumes are scaled down and the
-whole StorageOS cluster is restarted with a new version. Once the StorageOS
-cluster becomes usable, the applications are scaled up to their previous configuration.
+upgrade, StorageOS maintenance mode is enabled, the applications that use
```

Comment:
I think it is not clear that the operator will stop your apps. I think if we do scale down their deployments, the user must be warned energetically.